docs/01-app/03-building-your-application/07-configuring/03-environment-variables.mdx

```diff
@@ -178,6 +178,8 @@ const env = process.env
 setupAnalyticsService(env.NEXT_PUBLIC_ANALYTICS_ID)
 ```
 
+> **Note**: Bundled environment variables can also be set using the [`env`](/docs/app/api-reference/config/next-config-js/env) option in `next.config.js`. These do not require the `NEXT_PUBLIC_` prefix. (not recommended)
```

Comment:
The presence of `(not recommended)` makes me fee like we should omit this from the docs as it being included is a form of recommending.